src/index.html

```diff
@@ -17,6 +17,22 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href = "#" class="logo">
```

Comment:
There is an extra space around the `=` in the `href` attribute. It should be `href="#"` for consistency and to avoid potential issues.